src/Datadog.Trace/Configuration/TracerSettings.cs

```diff
@@ -270,6 +270,8 @@ public TracerSettings(IConfigurationSource source)
         /// </summary>
         public bool StartupDiagnosticLogEnabled { get; set; }
 
+        internal static bool DisableSharedInstance { get; set; }
```

Comment:
Can we add a comment describing intent of use?

Reply:
Sure!

Reply:
Just saw this. I would have suggested `SharedInstanceEnabled` to keep the `*Enabled` pattern. Luckily it's internal so we can change it anytime 😉